### DISCUSSION TITLE
How can i use ScreenSharing in cefSharp

### DISCUSSION THREAD

**Discussion by codersucks** [cefsharp/CefSharp#3716] [Q&A]
when i use cefsharp in  winform，i found the screensharing function of chromium does not work. dose cefsharp support  screensharing？ if not how can i achieve it.

**Comment by amaitland** [comment]
Only sharing the default monitor is currently supported

- https://github.com/cefsharp/CefSharp.MinimalExample/blob/cefsharp/91/CefSharp.MinimalExample.WinForms/Program.cs#L36
- https://bitbucket.org/chromiumembedded/cef/issues/2993/webrtc-based-screen-sharing-capturing-not

**Reply by codersucks** [reply]
thanks, will CefSharp gonna support app sharing?

**Reply by amaitland** [reply]
CEF would need to add support. Read the first few paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage for background.
